crates/adapters/dydx/tests/websocket.rs

```diff
@@ -618,16 +735,18 @@ async fn test_ping_pong() {
     let (addr, state) = start_test_server().await.unwrap();
     let ws_url = format!("ws://{addr}/v4/ws");
 
-    let mut client = DydxWebSocketClient::new_public(ws_url, Some(30));
+    let mut client = DydxWebSocketClient::new_public(ws_url, Some(1));
     client.connect().await.unwrap();
 
     wait_until_async(|| async { client.is_connected() }, Duration::from_secs(5)).await;
 
-    tokio::time::sleep(Duration::from_millis(500)).await;
+    wait_until_async(
+        || async { state.ping_count.load(Ordering::Relaxed) > 0 },
```

Comment:
It's good that we're making these tests more robust considering the low-spec runners in CI. But we have to be careful we don't lose the value of the tests in the process, i.e. just asserting > 0 is relaxing the condition a lot.